webapp/src/ts/main.ts

```diff
@@ -7,6 +7,22 @@ window.startupTimes.firstCodeExecution = performance.now();
 
 window.PouchDB = require('pouchdb-browser').default;
 window.$ = window.jQuery = require('jquery');
+window._phdcChanges = {
+  hierarchyDuplicatePrevention: {
+    health_center: {
+      ...Levenshtein,
+      props: [
+        {form_prop_path: `/data/health_center/name`, db_doc_ref: 'name'},
+        {form_prop_path: '/data/health_center/external_id', db_doc_ref: 'external_id'}
+      ],
+      queryParams: {
+        valuePaths: ['/data/health_center/is_user_flagged_duplicate', '/data/health_center/duplicate/action'],
+        // eslint-disable-next-line eqeqeq
```

Comment:
@jkuester Continuing to lobby for `!=` as the most excellent exception to the strict equality lint rule. Checks for both `null` and `undefined`.